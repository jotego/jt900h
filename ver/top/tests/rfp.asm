    ; control of the rfp
    main section code
    org 0
    ld wa,0x1234    ; common header
    ; actual test:
    ld wa,0x0000
    ld bc,0x0011
    ld de,0x0022
    ld hl,0x0033
    decf
    ld wa,0x3000
    ld bc,0x3011
    ld de,0x3022
    ld hl,0x3033
    decf
    ld wa,0x2000
    ld bc,0x2011
    ld de,0x2022
    ld hl,0x2033
    incf
    incf
    incf
    ld wa,0x1000
    ld bc,0x1011
    ld de,0x1022
    ld hl,0x1033
    nop
    nop
    nop
    nop
forever:
    jp forever
data:
    dw 0xcafe,0xbeef,0xffff,0xeeee,0xcccc
    end