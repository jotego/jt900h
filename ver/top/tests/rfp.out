00000000 - 00001000 - 00002000 - 00003000
00000011 - 00001011 - 00002011 - 00003011
00000022 - 00001022 - 00002022 - 00003022
00000033 - 00001033 - 00002033 - 00003033
00000000 - 00000000 - 00000000 - 00000000
